src/tuner/nccl_ofi_model.c

```diff
@@ -86,29 +86,26 @@ static const float nccl_base_lat[NCCL_NUM_ALGORITHMS][NCCL_NUM_PROTOCOLS] = {
 #define NCCL_OFI_TUNER_LL128_PROTO_BW_FACTOR      (0.9375)
 #define NCCL_OFI_TUNER_SIMPLE_PROTO_BW_FACTOR     (1.0)
 
-/* XXX: This is broadly incorrect. */
 static double all_reduce_ring_cost(struct nccl_ofi_tuner_model_dims const* dims,
 				   const int proto,
 				   const size_t nBytes)
 {
 	double base_latency = nccl_base_lat[NCCL_ALGO_RING][proto];
 	const double p2p_latency = nccl_nvlink_lat[NCCL_ALGO_RING][proto];
 
-	const double logranks = log(dims->num_ranks) / log(2);
-	const double logbytes = log(nBytes) / log(2);
-	const double lograt = (logbytes - logranks);
-	if (floor(lograt) < 23) {
-		/* Hack until we understand this better. */
-		return INFINITY;
-	}
+	const double logbytes = log(nBytes) * log(2);
```

Comment:
Why did this change from `/ log(2)` to `* log(2)`? The former seems correct to me, if you're trying to get to `log_2(nBytes)`.